subsys/usb/device_next/class/Kconfig.cdc_acm

```diff
@@ -21,6 +21,15 @@ config USBD_CDC_ACM_STACK_SIZE
 	help
 	  USB CDC ACM workqueue stack size.
 
+config USBD_CDC_ACM_SHORT_PACKET_TERMINATE_ENABLED
```

Comment:
Please come up with a concise option name.

Reply:
> Please come up with a concise option name.

Hi @jfischer-no, 

The naming "SHORT_PACKET_TERMINATE" was taken from WinUSB policy name [as documented here](https://learn.microsoft.com/en-us/windows-hardware/drivers/usbcon/winusb-functions-for-pipe-policy-modification). 

How about naming it `USBD_CDC_ACM_ENSURE_SHORT_PKT_FOR_LAST_TX_PKT`, which implies more directly that the option ensures that the last TX packet will be a short packet. 

What do you think?

Reply:
I have changed it to `USBD_CDC_ACM_ENSURE_SHORT_PKT_FOR_LAST_TX_PKT` in the latest commit push for now as I couldn't think of other better name for now.  Please let me know if you have any concern on the name.

Reply:
The new name is the opposite of concise. No?

Reply:
My bad, understand it wrongly for being not precise enough, but it seems like what you mean is to make it shorter but with the same meaning. What do you think about `USB_CDC_ACM_TX_ZLP_TERMINATE_ENABLED` ?

Reply:
Hi @jfischer-no, changed the config name to `USBD_CDC_ACM_TX_ZLP_ENABLED` and pushed the change. Please review, thank you.